packages/pg-cluster/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import {Queryable} from '@databases/pg';
+import Cluster from './Cluster';
+
```

Comment:
We should export the `Cluster` type so that it's possible to write a function that explicitly expects to be passed a `Cluster`.

```suggestion

export type {Cluster};

```